packages/cdk-api-gateway-domain-multi-region/README.md

```diff
@@ -0,0 +1,39 @@
+# @daysmart/cdk-api-gateway-domain-multi-region
```

Comment:
Can we add an extra note here explaining to a dev what resources are spun up as part of this? From my understanding, it's basically taking care of the custom domain portion of the serverless template from our normal set up.